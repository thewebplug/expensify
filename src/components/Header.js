import React from 'react';
import { NavLink } from 'react-router-dom';

const Header = () => (
    <header>
        <h1>Expensify</h1>
        <p><NavLink to="/" className={({isActive}) => isActive ? 'is-active' : undefined}>Home</NavLink></p>
        <p><NavLink to="create" className={({isActive}) => isActive ? 'is-active' : undefined}>Create</NavLink></p>
        <p><NavLink to="help" className={({isActive}) => isActive ? 'is-active' : undefined}>Help</NavLink></p>
    </header>
)

export default Header;