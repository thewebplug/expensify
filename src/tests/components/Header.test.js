import React from 'react';
import Header from '../../components/Header';
import {shallow} from 'enzyme';

test('should render header correctly', () => {
    const wrapper = shallow(<Header />);
    expect(wrapper).toMatchSnapshot();
})