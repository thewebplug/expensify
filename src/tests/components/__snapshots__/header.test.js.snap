// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render header correctly 1`] = `
<header>
  <h1>
    Expensify
  </h1>
  <p>
    <NavLink
      className={[Function]}
      to="/"
    >
      Home
    </NavLink>
  </p>
  <p>
    <NavLink
      className={[Function]}
      to="create"
    >
      Create
    </NavLink>
  </p>
  <p>
    <NavLink
      className={[Function]}
      to="help"
    >
      Help
    </NavLink>
  </p>
</header>
`;
